### PR TITLE
replacing pytensor-devs for aesara-devs

### LINKED ISSUE #6800: DOC: Address occurrences of `pytensor-devs`
### Issue with current documentation:

There are several occurrences of `pytensor-devs` in our docstrings. This was probably due to #6365 and some instances were overlooked.

https://github.com/pymc-devs/pymc/blob/7b08fc160bc07adcfcd10e135e727c681ffb1b77/pymc/sampling/jax.py#L118
https://github.com/pymc-devs/pymc/blob/7b08fc160bc07adcfcd10e135e727c681ffb1b77/pymc/logprob/rewriting.py#L421
https://github.com/pymc-devs/pymc/blob/7b08fc160bc07adcfcd10e135e727c681ffb1b77/pymc/distributions/shape_utils.py#L410
https://github.com/pymc-devs/pymc/blob/7b08fc160bc07adcfcd10e135e727c681ffb1b77/docs/source/learn/core_notebooks/pymc_pytensor.ipynb#L13
https://github.com/pymc-devs/pymc/blob/7b08fc160bc07adcfcd10e135e727c681ffb1b77/tests/sampling/test_jax.py#L137
https://github.com/pymc-devs/pymc/blob/7b08fc160bc07adcfcd10e135e727c681ffb1b77/tests/distributions/test_distribution.py#L695

### Idea or request for content:

All occurrences point to issues from Aesara and/or AePPL. The links should be rectified to the original issues that they refer to, unless we want to remove these altogether.

### PR DESCRIPTION
**Pull request by amyoshino**
**What is this PR about?**
Closes #6800 

This PR replaces occurrences of `pytensor-devs` for `aesara-devs` in the issues hyperlinks present in docstrings. It also fix a minor typo in the "What are value variables and why are they important?" section of the "PyMC and PyTensor" notebook.

What it is replacing?
`pytensor-devs/pytensor` -> `aesara-devs/aesara`
`pytensor-devs/aeppl` -> `aesara-devs/aeppl`

Tests:
- Hyperlinks are working
- `grep -r "pytensor-devs"` returns no records

**Checklist**
+ [X] Explain important implementation details 👆
+ [X] Make sure that [the pre-commit linting/style checks pass](https://docs.pymc.io/en/latest/contributing/python_style.html).
+ [X] Link relevant issues (preferably in [nice commit messages](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html))
+ [X] Are the changes covered by tests and docstrings?
+ [X] Fill out the short summary sections 👇

## Major / Breaking Changes
- ...

## New features
- ...

## Bugfixes
- ...

## Documentation
- ...

## Maintenance
- ...

----
:books: Documentation preview :books:: https://pymc--6817.org.readthedocs.build/en/6817/

### CODE REVIEW

File: docs/source/learn/core_notebooks/pymc_pytensor.ipynb [hunk 1]
```diff
@@ -10,10 +11,11 @@
     "\n",
     "**Authors:** [Ricardo Vieira](https://github.com/ricardoV94) and [Juan Orduz](https://juanitorduz.github.io/)\n",
     "\n",
-    "In this notebook we want to give an introduction of how PyMC models translate to PyTensor graphs. The purpose is not to give a detailed description of all [`pytensor`](https://github.com/pytensor-devs/pytensor)'s capabilities but rather focus on the main concepts to understand its connection with PyMC. For a more detailed description of the project please refer to the official documentation."
+    "In this notebook we want to give an introduction of how PyMC models translate to PyTensor graphs. The purpose is not to give a detailed description of all [`pytensor`](https://github.com/aesara-devs/aesara)'s capabilities but rather focus on the main concepts to understand its connection with PyMC. For a more detailed description of the project please refer to the official documentation."
```
Comment (larryshamalama) [inline docs/source/learn/core_notebooks/pymc_pytensor.ipynb]:
```suggestion
    "In this notebook we want to give an introduction of how PyMC models translate to PyTensor graphs. The purpose is not to give a detailed description of all [`pytensor`](https://github.com/pymc-devs/pytensor)'s capabilities but rather focus on the main concepts to understand its connection with PyMC. For a more detailed description of the project please refer to the official documentation."
```
My mistake in the original issue. We probably want this to point to the PyTensor repo url

**Review by larryshamalama — CHANGES_REQUESTED** [review]
Thanks @amyoshino for opening your PR :) One minor edit and we should be good to merge!

**Comment by amyoshino** [comment]
Amazing, thanks for the review! I just accepted the suggestion, let me know if something else is missing :)

**Review by larryshamalama — APPROVED** [review]
Looks good to me, thanks for your contribution! :)

**Pull request merged**